taskManager.js

```diff
@@ -13,11 +13,22 @@ function addTask(title, description, deadline) {
   dataStorage.saveTask(task);
 }
 
-function deleteTask(id) {
-    dataStorage.deleteTask(id);
+function deleteTask(title) {
+    dataStorage.deleteTask(title);
+  }
+
+function editTask(taskTitle, updatedTask) {
+    dataStorage.updateTask(taskTitle, updatedTask);
+}
```

Comment:
Consider renaming editTask to updateTask or rename the updateTask function in dataStorage.js